app/model/repository/AssignmentSolutionSubmissions.php

```diff
@@ -8,7 +8,7 @@
 use Doctrine\ORM\EntityManagerInterface;
 
 /**
- * @method AssignmentSolutionSubmission findOrThrow($id)
+ * @extends BaseRepository<AssignmentSolutionSubmission>
```

Comment:
What is this new develrye?

Reply:
Basically generics for PHP, phpstan supports this so we can easily type methods in base repository by applying extends on all child repositories

See https://phpstan.org/blog/generics-in-php-using-phpdocs